homeex/src/main/java/io/codelex/loops/practice/FizzBuzz.java

```diff
@@ -0,0 +1,32 @@
+package io.codelex.loops.practice;
+
+import java.util.Scanner;
+
+public class FizzBuzz {
+    public static void main(String[] args) {
+
+        System.out.print("Max value? : ");
+        Scanner in = new Scanner(System.in);
+        int n = in.nextInt();
+
+        for (int i = 1; i <= n; i++) {
```

Comment:
sanākuši duadz if, varētu noīsināt līdz kādiem 3-4 noteikti
uz to var drīzak paksatīties, ka sadalīt 3 daļās leitas kas japrintē, tikai vārdi, tikai atstarpes, jauna rinda, un apskatīt katru no tiem atsevišķi

Reply:
Kaut kā saīsināju bet laikam ne līdz galam